data-gathering/bot.py

```diff
@@ -14,48 +15,80 @@
 from datetime import datetime
 from query import store_articles, fetch_wikidata_info, get_id_and_subjects_and_grade
 
-# Open and read the JSON configuration file and access the configuration data as a dictionary
-with open("wikipedia_config.json", "r") as config_file:
-   wikipedia_config = json.load(config_file)
-
 
 # Accept 
 if len(sys.argv) > 2:
-   WIKIPEDIA_LANGUAGE  = sys.argv[1]
+   WIKIDATA_COUNTRY_ID  = sys.argv[1]
```

Comment:
"es", "en" are wikipedia languages
117, 77 are wikidata country ids

check these out:
https://www.wikidata.org/wiki/Q77 is uruguay
https://www.wikidata.org/wiki/Q117 is ghana

what I meant with my comment is the previous PR is that "CODE" is not a descriptive variable name, and I suggest it should be replaced by something along the lines of "WIKIDATA_COUNTRY_ID"

"WIKIPEDIA_LANGUAGE" is a good variable name

Reply:
oh, I get it
I've changed it

Reply:
oh, I get it now
I've updated the script.